src/librustdoc/clean/types.rs

```diff
@@ -120,8 +120,6 @@ crate struct Crate {
     crate module: Item,
     crate externs: Vec<ExternalCrate>,
     crate primitives: ThinVec<(DefId, PrimitiveType)>,
-    // These are later on moved into `CACHEKEY`, leaving the map empty.
-    // Only here so that they can be filtered through the rustdoc passes.
```

Comment:
Please leave the second line of the comment here; there's some quite tricky things going on around `external_traits`.